modules/virt-attaching-vm-to-ovn-secondary-nw-cli.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network.adoc              
+
+:_content-type: PROCEDURE                                             
+[id="virt-attaching-vm-to-ovn-secondary-nw-cli_{context}"]                                   
+= Attaching a virtual machine to an OVN-Kubernetes secondary network using the CLI
+
+You can connect a virtual machine (VM) to the OVN-Kubernetes secondary network by including the network details in the VM configuration.
+
+.Prerequisites
+* You have access to the cluster as a user with `cluster-admin` privileges.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+. Edit the `VirtualMachine` manifest to add the OVN-Kubernetes secondary network interface details, as in the following example:
++
+[source,yaml]
+----
+apiVersion: kubevirt.io/v1alpha3
```

Comment:
alpha3 is deprecated IIRC. 

We should use v1, like in this [example](https://docs.openshift.com/container-platform/4.13/virt/virtual_machines/virt-create-vms.html#virt-creating-vm-cli_virt-create-vms).